### DISCUSSION TITLE
Improve UI/UX for accessing Skin layout editor

### DISCUSSION THREAD

**Discussion by OfficialCRUGG** [ppy/osu#25555] [Ideas] [answered]
Soo... I've just spent like a solid ten minutes searching for the skin editor. As far as I know, the only way to access it currently is through the settings, which is extremely unintuitive in my opinion, especially since like *everything* points you towards something else.
- The main menu option of the beatmap editor being labelled "edit" (very unspecific) implies skin editing would be in there
- This is further amplified by the brush icon
- The beatmap editor has "osu!editor" in the top left, just like the skin editor (which one might have seen on YouTube or somewhere) which implies even more that editing skins is also possible from somewhere in this Screen

In case the skin editor can actually be accessed from somewhere within there, then it must be really hidden, because I've clicked through every menu option several times on my quest to finding the skin editor. I've actually only found it in the settings on coincidence after giving up and wanting to check something else there.

**Comment by peppy** [comment]
Sounds like a valid concern. Maybe after clicking "edit" on the main menu there should be a second level menu with "beatmap" and "skin" as options?

**Reply by peppy** [reply, marked as answer]
https://github.com/ppy/osu/pull/25561
